retag-for-jenkins

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ $# -lt 1 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 TAG"
```

Comment:
You could add a paragraph when the script comes in handy. Basically what you wrote in the commit message.

Reply:
Yes, thanks